### PR TITLE
update to peak driver 7.15.2 for kernels >4.0

### PR DESCRIPTION
**Pull request by frederikhegger**

### CODE REVIEW

**Comment by frederikhegger** [comment]
We have been working with the cob3-1 (14.04 + Indigo) intensively since one week and the new peak driver works without any problem under kernel 4.2.0-30-generic

**Comment by fmessmer** [comment]
Can someone test this on our r@w's? @ipa-mig @ipa-bnm?

**Comment by NadiaHG** [comment]
I already tested the peak driver 7.15 , it works, but it requires for Ubuntu14.04 an extra upgrade of the kernel.

**Comment by fmessmer** [comment]
If this new peak driver version is not usable with the default (up-to-date) kernel used with the LTS, but requires a manual kernel update, I'd prefer not to merge this....

However, if it's doable to add a decision which tar-ball to download based on the output of `uname -a` (or similar), we could support both versions....@frederikhegger @ipa-nhg

File: libpcan/install_pcan.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SOURCE_DIR=build/peak-linux-driver-7.9
+SOURCE_DIR=build/peak-linux-driver-7.15.2
 KERNEL_VER=/usr/src/linux-headers-"`uname -r`"
```
Comment (fmessmer) [inline libpcan/install_pcan.sh:2]: can we use this for making the decision?

**Comment by fmessmer** [comment]
would cherry-picking #74 to `indigo_dev` supersede this one?
@ipa-bnm @frederikhegger @ipa-nhg

**Comment by fmessmer** [comment]
@ipa-bnm 
Could you please provide a cherry-pick PR for indigo_dev, then?

**Comment by benmaidel** [comment]
@ipa-fxm oh, I thought you'll do that. But sure I will!

**Comment by fmessmer** [comment]
done in #78

**Pull request closed without merging**